test/cmdlineTests/standard_ewasm_requested/output.json

```diff
@@ -146,19 +144,5 @@
     )
 )
 
-(func $revert
-    (param $x1 i64)
-    (param $x2 i64)
-    (param $x3 i64)
-    (param $x4 i64)
-    (param $y1 i64)
-    (param $y2 i64)
-    (param $y3 i64)
-    (param $y4 i64)
-    (block $label__8
-        (call $eth.revert (call $to_internal_i32ptr (local.get $x1) (local.get $x2) (local.get $x3) (local.get $x4)) (call $u256_to_i32 (local.get $y1) (local.get $y2) (local.get $y3) (local.get $y4)))
-    )
-)
-
```

Comment:
Why does this disappear after your changes? I get that the optimizer might just decide to inline it but I don't see anything in the PR that would change optimization logic. Is this just a random optimization decision that ended up being different because some instructions got shuffled around or is there more to it?

Reply:
Hmm.. I don't know why this exactly happened here. For me it looks like a random optimizer decision, but I don't have an explanation why this really happened.

Reply:
Not sure. Do you have a diff when optimizer is not enabled?

Reply:
@aarlt Could you get a diff of that?

Reply:
Looks like this a plausible change after all. https://github.com/ethereum/solidity/pull/10406#discussion_r564523906